apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/notes-panel.e2e.cy.ts

```diff
@@ -0,0 +1,80 @@
+describe(`open self notes panel for resource`, () => {
+    const photographBaseRoute = `/Resources/Photographs/`;
+
+    const photographId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002`;
+
+    describe(`when opening the self notes panel for a resource`, () => {
+        beforeEach(() => {
+            cy.visit(`${photographBaseRoute}${photographId}`);
+        });
+
+        describe(`when the photograph detail full view has loaded`, () => {
+            it(`should expose the open note panel button`, () => {
+                cy.getByDataAttribute('TextSnippetIcon').should('exist');
```

Comment:
Use a more declarative data attribute. Something like `self-notes-panel-expander-button` would be better.

Reply:
I gave the button the name open-notes-panel-button in categorizable-page-layout.tsx but then didn't carry it through to the cypress test :(